src/attachments/attachments.spec.ts

```diff
@@ -0,0 +1,304 @@
+import type { ErrorResponse } from '../interfaces';
+import { Resend } from '../resend';
+
+const resend = new Resend('re_zKa4RCko_Lhm9ost2YjNCctnPjbLw8Nop');
+
+describe('Attachments', () => {
+  afterEach(() => fetchMock.resetMocks());
+
+  describe('get', () => {
+    describe('when attachment not found', () => {
+      it('returns error', async () => {
+        const response: ErrorResponse = {
+          name: 'not_found',
+          message: 'Attachment not found',
+        };
+
+        fetchMock.mockOnce(JSON.stringify(response), {
+          status: 404,
+          headers: {
+            'content-type': 'application/json',
+            Authorization: 'Bearer re_zKa4RCko_Lhm9ost2YjNCctnPjbLw8Nop',
+          },
+        });
+
+        const result = await resend.attachments.get({
+          emailId: '61cda979-919d-4b9d-9638-c148b93ff410',
```

Comment:
In the spirit of keeping everything isolated during the alpha testing, and not causing confusion with outbound, how about we rename `emailId` to `inboundId`?